control/control.yaml

```diff
@@ -213,3 +213,28 @@ components:
         lacp_member_state:
           $ref: './lacp.yaml#/components/schemas/LacpMember.State'
           x-field-uid: 2
+    Bgp.GraceFulRestart:
+      description: >-
+        Initiates BGP Graceful Restart process.
+      type: object
+      required: [restart_time]
+      properties:
+        peer_names:
+          description: >-
+            The names of device BGP peer objects to control.
+            If no names are specified then all BGP peer objects that match the
+            x-constraint will be affected.
+          type: array
+          items:
+            type: string
+          x-constraint:
+          - '/components/schemas/Device.Bgp/properties/name'
+          x-field-uid: 1
+        restart_time:
+          description: >-
+            This is the estimated time (in seconds) it will take for the
```

Comment:
This is not what is going in the BGP PDU but rather the time after which selected the BGP peer(s) will restart the connection with Restart Bit set in GR Capability to emulate Graceful Restart. The max value might be better to have as something like 1 hr for LLGR use-cases for example.